### PR TITLE
new icon: arduino (original, plain, original-wordmark, plain-wordmark)

### PR DESCRIPTION
**Pull request by Kartstig**
- Adds the arduino icon

### CODE REVIEW

**Comment by Thomas-Boi** [comment]
Hello there,
Recently, we just introduced an automation pipeline to Devicon which will help us create the icons. Before we can merge your changes in, there are a few things that we need to do first:

1. Merge the current devicon/develop branch into your PR branch. This contains the build script needed for us to build your icons.
2. Read the `CONTRIBUTING.md` for things to do before merging and review our icon standards.
3. Push your changes and/or branches. We will review them once this is done and merge your PR if everything goes well.


Thank you for contributing to this repo. We are sorry that we haven't accept any PR for awhile. Now that we have a build pipeline, we can merge PRs more quickly than before. 

We hope to see your icons in the Devicon repo soon :)

Cheers,
Devicon Team

**Comment by Thomas-Boi** [comment]
Hello there,

We notice that this PR has been inactive for awhile now. It would be a waste of your time and effort if we just close this PR and throw away the SVGs in here. Thus, 1 month from now, if this PR remains inactive/stale, we will fork your repo, fix any issues, and merge your icons ourselves.

Since GitHub tracks contributions by commits, you will still be credited. We also hope that you will help us merge this PR before the deadline comes.

Cheers,
Devicon Team

**Comment by Kartstig** [comment]
- [x] Changed the base
- [x] Set devicon.json
- [x] Updated the svgs

File: icons/arduino/arduino-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="17.197916mm"
+   height="12.273544mm"
+   viewBox="0 0 17.197916 12.273544"
```
Comment (amacado) [inline icons/arduino/arduino-original-wordmark.svg:12]: viewBox, height and width does not fulfill our requirements (0 0 128 128).

**Review by amacado — CHANGES_REQUESTED** [review]
Please have a look at our guidelines regarding [svg requirements](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards). The when you update this pull request, those requirements should be checked by out bot.. Let's see :)

**Pull request merged**